src/server/handlers/subnet.rs

```diff
@@ -0,0 +1,46 @@
+//! The shared subnet manager module for all subnet management related RPC method calls.
+
+use anyhow::{anyhow, Result};
+use std::collections::HashMap;
+use crate::config::Subnet;
+use crate::jsonrpc::JsonRpcClient;
+use crate::manager::LotusSubnetManager;
+
+pub(crate) struct ManagerConfigBundle<T: JsonRpcClient> {
```

Comment:
I would embed all of this as part of the pool/manager object to avoid having it independently.